node/src/database/db_migrations.rs

```diff
@@ -200,6 +207,41 @@ impl MigDeclarationUtilities for MigDeclarationUtilitiesReal<'_> {
     }
 }
 
+trait StatementObject: Display {
```

Comment:
You opined that these are no longer necessary. If you choose to back them out, I'm okay with that; if you choose to leave them in, I'm almost as okay with that. Your codebase, your choice.